File: GGAndroid_2.0/GGDemo/src/main/java/io/ganguo/demo/view/dialog/DemoBottomDialog.java
package io.ganguo.demo.view.dialog;


import android.content.Context;

import io.ganguo.demo.viewmodel.dialog.DialogDemoBottomVModel;
import io.ganguo.viewmodel.databinding.DialogGgBinding;
import io.ganguo.viewmodel.ui.dialog.BottomVModelDialog;

/**
 * <p>
 * 底部弹窗Demo - Dialog
 * </p>
 * Created by leo on 2018/9/30.
 */
public class DemoBottomDialog extends BottomVModelDialog<DialogGgBinding, DialogDemoBottomVModel> {
    public DemoBottomDialog(Context context) {
        super(context);
    }

    @Override
    public DialogDemoBottomVModel createViewModel() {
        return new DialogDemoBottomVModel();
    }

    @Override
    public void onViewAttached(DialogDemoBottomVModel viewModel) {

    }
}

File: GGAndroid_3.0/main/app-mvvm-v1-sample/src/main/java/io/ganguo/demo/view/dialog/DemoBottomDialog.kt
package io.ganguo.demo.view.dialog


import android.content.Context
import io.ganguo.core.R

import io.ganguo.demo.viewmodel.dialog.DialogDemoBottomVModel
import io.ganguo.viewmodel.core.BaseViewModel
import io.ganguo.viewmodel.databinding.DialogGgBinding
import io.ganguo.viewmodel.pack.ui.dialog.BottomVModelDialog

/**
 *
 *
 * 底部弹窗Demo - Dialog
 *
 * Created by leo on 2018/9/30.
 */
class DemoBottomDialog(context: Context,theme: Int= R.style.Dialog_Immersion) : BottomVModelDialog<DialogGgBinding, DialogDemoBottomVModel>(context, theme) {

    override fun createViewModel(): DialogDemoBottomVModel {
        return DialogDemoBottomVModel()
    }


    override fun onViewAttached(viewModel: DialogDemoBottomVModel) {
    }
}
